pkg/reconciler/identity/resources/iam_policy_bindings.go

```diff
@@ -25,23 +25,28 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	Add    = "add"
+	Remove = "remove"
+)
+
 // AddIamPolicyBinding will add iam policy binding, which is related to a provided k8s ServiceAccount, to a GCP ServiceAccount.
-func AddIamPolicyBinding(ctx context.Context, projectID string, gServiceAccount *string, kServiceAccount *corev1.ServiceAccount) error {
-	if err := SetIamPolicy(ctx, "add", projectID, gServiceAccount, kServiceAccount); err != nil {
-		return fmt.Errorf("failed to add iam policy binding: %w", err)
+func AddIamPolicyBinding(ctx context.Context, projectID, gServiceAccount string, kServiceAccount *corev1.ServiceAccount) error {
+	if err := setIamPolicy(ctx, Add, projectID, gServiceAccount, kServiceAccount); err != nil {
+		return err
 	}
 	return nil
 }
 
 // RemoveIamPolicyBinding will remove iam policy binding, which is related to a provided k8s ServiceAccount, from a GCP ServiceAccount.
-func RemoveIamPolicyBinding(ctx context.Context, projectID string, gServiceAccount *string, kServiceAccount *corev1.ServiceAccount) error {
-	if err := SetIamPolicy(ctx, "remove", projectID, gServiceAccount, kServiceAccount); err != nil {
-		return fmt.Errorf("failed to remove iam policy binding: %w", err)
+func RemoveIamPolicyBinding(ctx context.Context, projectID, gServiceAccount string, kServiceAccount *corev1.ServiceAccount) error {
+	if err := setIamPolicy(ctx, Remove, projectID, gServiceAccount, kServiceAccount); err != nil {
+		return err
 	}
 	return nil
 }
 
-func SetIamPolicy(ctx context.Context, action, projectID string, gServiceAccount *string, kServiceAccount *corev1.ServiceAccount) error {
+func setIamPolicy(ctx context.Context, action, projectID string, gServiceAccount string, kServiceAccount *corev1.ServiceAccount) error {
 	iamService, err := iam.NewService(ctx)
```

Comment:
I think I saw you created an issue already. Can you add a comment here saying something like:

`// TODO This should be injectable so that we can unit test it <link to the issue> `